lib/bundler.js

```diff
@@ -92,6 +92,14 @@ function bundle(options) {
     cwd: tempDir,
     stdio: "inherit"
   });
+  if (config.npmPackages) {
+    for(const dependency of config.npmPackages) {
+      Execa.sync("meteor", ["npm", "install", dependency], {
```

Comment:
Please install using a single command execution. It is much much more efficient.

    npm install a b c...

Reply:
Also, in case the `npmPackages` is an empty array, that command I suggested above can work unexpectedly. So be sure to make the necessary validations before hand